test/factories/users.rb

```diff
@@ -0,0 +1,6 @@
+FactoryGirl.define do
+  factory :valid_user, class: User do
+    email 'some_user@gmail.com'
+    password 'some_password'
```

Comment:
@vlasiak I suggest to imagine that there is a real user, not just "some user". For example `steve@tada.com`. Password as well could be more realistic (or similar to auto generated), for example: `q#5&j0b`.